pkg/schedule/windowed_schedule_test.go

```diff
@@ -50,26 +94,28 @@ func TestWindowedSchedule(t *testing.T) {
 					time.Sleep(w.Interval * 2)
 				}
 			}
-			// we should have either 16 or 17 minus 3 missed
-			So(len(r), ShouldBeBetweenOrEqual, 15, 17)
+			// we should have either 17 or 19 minus 4 missed
```

Comment:
The comment should read, "We should have results between x and y".

This test comes back with 15 around 10% of the time for me.

![img](https://www.evernote.com/l/AA3kfXNVWshH_q7dO08dcEYgwcNp9ymgYHoB/image.png)